docs/source/dev/supported_vrplib_fields.rst

```diff
@@ -55,6 +55,10 @@ PyVRP supports the following data sections:
 .. glossary::
    :sorted:
 
+   ``BACKHAUL_SECTION``
+      Array of backhaul quantities, one for each location.
+      The depot locations *must* have a backhaul quantity of zero.
+
    ``DEMAND_SECTION``
       Array of demands, one for each location.
       The depot locations *must* have zero demand.
```

Comment:
Maybe make the difference between backhaul (client back to depot) and demand/linehaul (depot to client) a bit clearer. Some thing like

```suggestion
   ``BACKHAUL_SECTION``
      Array of backhaul quantities, one for each location.
      This is the amount picked up at the location and transported back to the depot.
      The depot locations *must* have a backhaul quantity of zero.

   ``DEMAND_SECTION``
      Array of demands, one for each location.
      This is the delivery amount from the depot to the client.
      The depot locations *must* have zero demand.
```

Additionally, I think all references to depot locations needing to have zero values (here, and in various other places) can be removed - the `Depot` type no longer gets those arguments anyway. I missed updatting this when separating the structs.